format/spec.md

```diff
@@ -298,16 +298,101 @@ Iceberg tables must not use field ids greater than 2147483447 (`Integer.MAX_VALU
 
 The set of metadata columns is:
 
-| Field id, name              | Type          | Description |
```

Comment:
Any chance we can avoid unnecessary changes? I usually use a text editor. I believe there are [ways to disable auto-formatting](https://youtrack.jetbrains.com/issue/IJPL-92295/Markdown-Add-option-to-disable-smart-tables-introduced-in-IntelliJ-IDEA-2021.3) in IntelliJ as well.